modules/bridge/src/main/ts/ephox/bridge/api/Menu.ts

```diff
@@ -42,5 +43,12 @@ export {
   ContextMenuApi,
   ContextMenuContents,
   ContextMenuItem,
-  ContextSubMenu
+  ContextSubMenu,
+
+  createCardMenuItem,
+  CardMenuItem,
+  CardMenuItemSpec,
+  CardMenuItemInstanceApi,
+  ContainerItemSpec,
+  ContainerItem
```

Comment:
Should this have all the other container types exposed? As they can be used, so I believe the types should be exposed.